rv-monitor-rt/src/main/java/com/runtimeverification/rvmonitor/java/rt/util/TraceUtil.java

```
@@ -28,6 +34,35 @@ public static Integer getShortLocation(String fullLOC) {
         return shortLocation;
     }
 
+    /**
+     * Updates locationMap and freshID based on a potentially existing locationMap recorded in locationMapFile.
+     * The method will silently return if locationMapFile is empty or does not exist.
```

silently return --> return immediately